tests/inputFields.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { test, expect } from "@playwright/test";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/");
+
+  await page.getByText(" Pet Types").click();
+  const tableTitle = await page.locator("h2").textContent();
+  expect(tableTitle).toEqual("Pet Types");
+});
+
+test("Update pet type", async ({ page }) => {
+  const tableRows = page.locator("tbody tr");
+  await page.waitForSelector('[id="0"]');
+  for (let row of await tableRows.all()) {
+    const cellValue = await row.locator("input").inputValue();
+    if (cellValue === "cat") {
+      await row.getByRole("button", { name: "Edit" }).click();
+      break;
+    }
+  }
+
+  await expect(page.locator("h2")).toHaveText("Edit Pet Type");
+
+  const petTypeInput = page.locator("#name");
+  await petTypeInput.click();
+  await petTypeInput.clear();
+
+  //   await page.waitForTimeout(5000);
+  await petTypeInput.fill("rabbit");
+  await page.getByRole("button", { name: "Update" }).click();
+
+  await page.waitForSelector('[id="0"]');
```

Comment:
This wait can be safely removed since in the next line you have the same element that Playwright will automatically wait for.

Reply:
good point, thank you